### ISSUE TITLE
Integrate doc/legacy into sphinx docs

### ISSUE THREAD

**Issue by nicolan** [cyrusimap/cyrus-imapd#1769]
- [ ] ag.html
- [ ] anoncvs.html
- [ ] bugs.html
- [ ] changes.html
- [ ] faq.html
- [ ] feedback.html
- [ ] index.html
- [ ] install-admin-mb.html
- [ ] install-auth.html
- [x] install-backups.html
- [ ] install-compile.html
- [ ] install-configure.html
- [ ] install-http.html
- [ ] install-murder.html
- [ ] install-netnews.html
- [ ] install-perf.html
- [ ] install-prereq.html
- [ ] install-replication.html
- [ ] install-sieve.html
- [ ] install-snmpmon.html
- [ ] install-testing.html
- [ ] install-upgrade.html
- [ ] install-virtdomains.html
- [ ] install.html
- [ ] mailing-list.html
- [ ] man.html
- [ ] murder.fig
- [ ] netnews.fig
- [ ] os.html
- [ ] overview.html
- [ ] questions.html
- [ ] readme.html
- [ ] sieve-protocol.html
- [ ] sieve.html
- [ ] specs.html

**Comment by morrme** [comment]
What does this require?

**Comment by elliefm** [comment]
@morrme:

* identify legacy docs which have already been integrated into RST (such as install-backups.html)
* identify information that the legacy docs contain, but which the RST docs do not
* determine whether it's still relevant/correct
* integrate it into the RST docs

(Some of this work has already been done, but the record of which files have been reviewed/dealt with already was lost with the death of one of our previous bug tracking systems :(  So we started the list again to make sure nothing was missed.)

**Comment by nicolan** [comment]
**install-compile**: I'm tempted to remove this altogether as we have new compile docs already in sphinx. While it does outline the options to `configure`, I'm not sure this needs to be converted as this means more content that needs to be kept in sync with the code, and hopefully anyone who needs non-standard options to configure already will think to look there.

@elliefm What do you think? Convert or drop doc/legacy/install-compile.html?

**Comment by elliefm** [comment]
Yeah I'm happy to lose that file, it doesn't offer anything particularly unique and has a bunch of date or less than ideal instructions in it anyway

**Comment by rjbs** [comment]
:robot: I'm closing this issue by automation.

This issue has been sitting untouched for over seven years.  We're trying to get a good handle on our open issues and pull requests, so that we can reliably use them as a backlog of things we hope to work on.  If you are confident that this issue should be reconsidered, reopen it and explain why.

Remember, though, that there are only so many developers working on Cyrus, and we'd like to focus on the most important and/or low-hanging work.

Before you reopen this issue, please check that this issue is still present in the latest release of Cyrus, or the main development branch. An issue is considerably more likely to get resolved if you can provide a pull request which reproduces the issue in a Cassandane test.
